src/server/services/buurt/buurt.ts

```diff
@@ -279,7 +279,11 @@ export async function loadFeatureDetail(
     );
   }
 
-  const response = await requestData(requestConfig, requestID);
+  if (config.requestConfig?.request) {
+    requestConfig.request = config.requestConfig.request;
+  }
+
+  const response: any = await requestData(requestConfig, requestID);
```

Comment:
```suggestion
  const response = await requestData(requestConfig, requestID);
```
any is niet nodig,
![Screenshot 2024-02-05 at 13 45 31](https://github.com/Amsterdam/mijn-amsterdam-frontend/assets/1895171/18707c74-c791-402d-b9f3-257222a6ea67)